pkg/exchange/okex/parse.go

```diff
@@ -306,26 +320,30 @@ func parseOrder(v *fastjson.Value) ([]okexapi.OrderDetails, error) {
 	return orderDetails, nil
 }
 
-func parseData(v *fastjson.Value) (interface{}, error) {
+func parseData(v *fastjson.Value) (interface{}, string, error) {
 
 	channel := string(v.GetStringBytes("arg", "channel"))
 
 	switch channel {
+	case "books5":
+		data, err := parseBookData(v)
```

Comment:
I think you can include the channel in the BookData struct? so that we can keep this parse function simple.

Reply:
ok